### PR TITLE
Update changelog and version for 0.19.0

### PR DESCRIPTION
**Pull request by radhikaj**
Prep for v0.19.0 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,23 +9,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### Added
 
 - OE SDK can now be built with Clang-11 and it is recommended to upgrade the compiler to Clang-11 if you are building the SDK from source. Building with Clang-10 is still supported until the next Clang upgrade, but is not recommended.
-
-- Building OE SDK now includes the following LVI mitigation options:
-  - `ControlFlow-GNU` - enables LVI mitigation using the existing GNU-based mitigation specified by `LVI_MITIGATION_BINDIR`. This option is recommended when building OE SDK with Clang-10.
-  - `ControlFlow-Clang` - enables Clang-based LVI mitigations. Choosing this option requires Clang-11.
-  - `ControlFlow` - enables LVI mitigation but default to the recommended method, which is currently ControlFlow-GNU.
-  - `None` - no LVI mitigations are enabled.
+  - Building OE SDK now includes the following LVI mitigation options:
```
Comment (radhikaj) [inline CHANGELOG.md:18]:
```suggestion
  - Building OE SDK enclaves now includes the following LVI mitigation options:
```

**Review by anakrish — APPROVED** [review]
LGTM

Have the changes to help with debugging (e.g abort backtrace) been published as part of an official OE release yet?

**Review by asvrada — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -50,12 +44,45 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- snmalloc (0.6.0) now only requires 16KiB initially per thread, compared to 256 KiB previously (0.5.3). Thread-local heaps only grow by a constant amount until the next power of 2, eg. 16KiB, 16KiB, 32KiB, 64KiB, ... 1MiB, 2MiB, 2MiB, ..., as opposed to a fixed amount previously (256 KiB). Allocator metadata is kept separately, making corruption attacks less likely. There are some applications for which this update in snmalloc will result in an increase in EPC memory as specified in the enclave configuration file.
+
+
+[v0.18.5][v0.18.5_log]
+--------------
+### Security
+- Updated OpenSSL used inside the enclave to v1.1.1t. See [OpenSSL's release notes](https://www.openssl.org/news/openssl-1.1.1-notes.html) for more details
+
+
+[v0.18.3][v0.18.3_log]
```
Comment (CyanDevs) [inline CHANGELOG.md:59]: We have v0.18.4 but not v0.18.3
```suggestion
[v0.18.4][v0.18.4_log]
```

**Review by CyanDevs — APPROVED** [review]
Please fix the v0.18.4 version. Everything else looks good.

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,23 +9,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-  - `ControlFlow-GNU` - enables LVI mitigation using the existing GNU-based mitigation specified by `LVI_MITIGATION_BINDIR`. This option is recommended when building OE SDK with Clang-10.
-  - `ControlFlow-Clang` - enables Clang-based LVI mitigations. Choosing this option requires Clang-11.
-  - `ControlFlow` - enables LVI mitigation but default to the recommended method, which is currently ControlFlow-GNU.
-  - `None` - no LVI mitigations are enabled.
+- OE SDK can now be built with Clang-11 and it is recommended to upgrade the compiler to Clang-11 if you are building the SDK from source. 
+  - Building OE SDK enclaves now includes the following LVI mitigation options:
+    - `ControlFlow-GNU` - enables LVI mitigation using the existing GNU-based mitigation specified by `LVI_MITIGATION_BINDIR`. This option is recommended when building OE SDK with Clang-10.
+    - `ControlFlow-Clang` - enables Clang-based LVI mitigations. Choosing this option requires Clang-11.
+    - `ControlFlow` - enables LVI mitigation but default to the recommended method, which is currently ControlFlow-GNU.
+    - `None` - no LVI mitigations are enabled.
```
Comment (radhikaj) [inline CHANGELOG.md:22]:
```suggestion
    - `None` - no LVI mitigations are enabled.
    - *Note*: The packages released in 0.19.0 are built with ControlFlow-GNU mitigations
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -50,12 +44,45 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
   See `openenclave/attestation/tdx/evidence.h` for more detail of the APIs.
 
-## Changed
-- Fix the incorrect behavior of pthread_mutex_init() and std::mutex such that they no longer create a recursive lock by default.
-- `oe_verify_report` now reports the verification result in the `parsed_report` output (only valid when the API returns OE_OK or OE_TCB_LEVEL_INVALID). For the SGX case, the definition of the result is based on `oe_sgx_tcb_status_t`.
+- Added support for V4 collateral from Intel for SGX and TDX
+
+### Changed
+- snmalloc (0.6.0) now only requires 16KiB initially per thread, compared to 256 KiB previously (0.5.3). Thread-local heaps only grow by a constant amount until the next power of 2, eg. 16KiB, 16KiB, 32KiB, 64KiB, ... 1MiB, 2MiB, 2MiB, ..., as opposed to a fixed amount previously (256 KiB). Allocator metadata is kept separately, making corruption attacks less likely. There are some applications for which this update in snmalloc will result in an increase in EPC memory as specified in the enclave configuration file.
+
```
Comment (radhikaj) [inline CHANGELOG.md:51]: - Open Enclave SDK is no longer tested on Ubuntu 18.04. Ubuntu 18.04 is at EOL. v0.19.0 will not contain packages for Ubuntu 18.04.

File: CHANGELOG.md [hunk 5]
```diff
@@ -50,12 +45,45 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
```
Comment (radhikaj) [inline CHANGELOG.md:52]:
```suggestion
- Open Enclave SDK is no longer tested on Ubuntu 18.04. Ubuntu 18.04 is at EOL. v0.19.0 will not contain packages for Ubuntu 18.04.
```

**Comment by radhikaj** [comment]
> Please fix the v0.18.4 version. Everything else looks good.

thanks for pointing this out @CyanDevs

**Comment by CyanDevs** [comment]
bors r-

An expired GitHub token caused Jenkins to not start bors builds. This has been fixed but there's also no need to run tests on this change. Will add `VERSION` to the list of files to skip in another PR.

**Pull request merged**